resources/views/beatmapset_watches/index.blade.php

```diff
@@ -109,7 +109,7 @@ class="beatmapset-watches__cover"
                 @endif
             </table>
 
-            @include('objects._pagination_v0', ['object' => $watches])
+            @include('objects._pagination_v0', ['object' => $watches, 'modifiers' => ['light-bg']])
```

Comment:
the page seems to be dark?

![](https://s.myconan.net/2019-07/firefox_2019-07-18_15-13-05.png?)